### PR TITLE
i18n(fr): translate middleware-cant-be-loaded.mdx

### PR DESCRIPTION
**Pull request by MartinFerret**
French translation 😀

### CODE REVIEW

File: src/content/docs/fr/reference/errors/middleware-cant-be-loaded.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Impossible de charger le middleware.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+
+> **MiddlewareCantBeLoaded**: Le middleware a généré une erreur pendant qu'Astro essayait de le charger.
```
Comment (VoxelMC) [inline src/content/docs/fr/reference/errors/middleware-cant-be-loaded.mdx:10]: I am not sure, as I have never developed in French, let alone another language before. When the error is thrown, I - however naively - assume it isn't translated.

Should this part be translated, @sarah11918? Or would it be more helpful to keep it this way?

**Review by VoxelMC — COMMENTED** [review]
Thank you again, Martin! I only left one suggestion for a line removal and a question for the Maintainers about whether or not to translate the error message itself.

Other than that, LGTM from a non-native speaker!

File: src/content/docs/fr/reference/errors/middleware-cant-be-loaded.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Impossible de charger le middleware.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **MiddlewareCantBeLoaded**: Le middleware a généré une erreur pendant qu'Astro essayait de le charger.
```
Comment (sarah11918) [inline src/content/docs/fr/reference/errors/middleware-cant-be-loaded.mdx:7]:
```suggestion
> **MiddlewareCantBeLoaded**: The middleware threw an error while Astro was trying to loading it.
```
Correct! This part should be in English, exactly as the error message would be seen. Everything else *is* translated!

**Comment by sarah11918** [comment]
Also, I believe that this error will have to be added to the main error reference page: https://docs-git-fork-martinferret-patch-3-astrodotbuild.vercel.app/fr/reference/error-reference/

File: src/content/docs/fr/reference/errors/middleware-cant-be-loaded.mdx (hunk 2 above)
Comment (MartinFerret) [reply src/content/docs/fr/reference/errors/middleware-cant-be-loaded.mdx]: I've juste made the change

File: src/content/docs/fr/reference/errors/middleware-cant-be-loaded.mdx (hunk 1 above)
Comment (thomasbnt) [reply src/content/docs/fr/reference/errors/middleware-cant-be-loaded.mdx]: Hello @VoxelMC ! This error code must not be translated yep!

https://github.com/withastro/docs/blob/main/src/content/docs/en/reference/errors/middleware-cant-be-loaded.mdx

**Comment by sarah11918** [comment]
SVP, review the current version after resolving conflicts!

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Pull request merged**